modules/mono/csharp_script.cpp

```diff
@@ -1353,6 +1355,7 @@ void CSharpLanguage::_editor_init_callback() {
 
 	// Enable it as a plugin
 	EditorNode::add_editor_plugin(godotsharp_editor);
+	ED_SHORTCUT("mono/build_solution", TTR("Build solution"), KEY_MASK_ALT | KEY_B);
```

Comment:
```suggestion
	ED_SHORTCUT("mono/build_solution", TTR("Build Solution"), KEY_MASK_ALT | KEY_B);
```

Reply:
Hi @akien-mga, originally I had put the wording as `"Build Mono Solution"` there, but then the tooltip on the button looked ugly as it doubled up the text as the shortcut text was different from the button tooltip text:

![image](https://user-images.githubusercontent.com/233380/133253915-23be1ca7-c37a-479f-9bcd-8fe2fb28eafb.png)

I changed it to match the button's text, hence `"Build solution"`, which simplifies the tooltip by merging the two texts:

![image](https://user-images.githubusercontent.com/233380/133254004-c6638f2d-8716-4226-bd0f-def24230f7a5.png)

So I'd suggest that both labels need to be changed to be consistent if the casing is desired:

https://github.com/godotengine/godot/blob/5e6f4114d4b31f8d21e0ebe93321cc79718c0ba3/modules/mono/editor/GodotTools/GodotTools/GodotSharpEditor.cs#L426

(Unsure how I can add my own "suggestion" in response, but let me know, and I'll make that change.)